Test/Unit/BoltTestCase.php

```diff
@@ -3,13 +3,318 @@
 namespace Bolt\Boltpay\Test\Unit;
 
 use PHPUnit\Framework\TestCase;
+use ReflectionClass;
+use ReflectionException;
+use ReflectionObject;
+use ReflectionProperty;
+use PHPUnit\Framework\Constraint\IsType;
 
-class BoltTestCase extends TestCase
-{
-    protected function skipTestInUnitTestsFlow()
+if (!defined('PHP_CURRENT_VERSION')) {
+    define('PHP_CURRENT_VERSION', (float)phpversion());
+}
+
+if (PHP_CURRENT_VERSION < 7.2) {
```

Comment:
What's the logic behind this?
PHPUnit 9 requires PHP >= 7.3
Magento2.4 supports PHP 7.4 (eventually 7.3 but not tested or recommended)